Lib/glyphsLib/builder/transformations/intermediate_layers.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import uuid
+
+from fontTools.varLib.models import VariationModel, normalizeValue
+
+from glyphsLib.classes import GSLayer, GSNode, GSPath
+from glyphsLib.builder.axes import get_regular_master
+
+
+logger = logging.getLogger(__name__)
+
+
+def resolve_intermediate_components(font):
+    components_with_intermediate_layers = set()
+    all_intermediate_locations = set()
```

Comment:
you don't seem to be currently using these two sets. In the comment below you say "Later we _will_ check if...", do you mean these sets that you're populating will be useful in the future when we actually write that logic (like a TODO or self reminder)?

Reply:
I wrote some code to try to fix #954, but because I couldn't replicate that issue I removed it again. Or most of it, at least. These will be leftovers.